Minio/RequestExtensions.cs

```diff
@@ -96,7 +96,16 @@ public static Task WrapperPutAsync(this IMinioClient minioClient, string url, St
             var response = await minioClient.Config.HttpClient.SendAsync(request,
                     HttpCompletionOption.ResponseHeadersRead, cancellationToken)
                 .ConfigureAwait(false);
-            responseResult = new ResponseResult(request, response);
+            var statusCode = response.StatusCode;
+            var memoryStream = new MemoryStream();
+            if (statusCode == HttpStatusCode.OK)
+            {
+                var stream = await response.Content.ReadAsStreamAsync().ConfigureAwait(false);
+                await stream.CopyToAsync(memoryStream).ConfigureAwait(false);
+                memoryStream.Position = 0;
+            }
+
+            responseResult = new ResponseResult(request, response, memoryStream);
```

Comment:
Because of the previous review comments, this part needs to be cleaned up as `responseResult` has the `ContentStream` is already set up.

Reply:
You should also run `dotnet regitlint` for formatting problems.